### PR TITLE
Fixed an issue with oversized table headers

### PR DESCRIPTION
**Pull request by vxsx**
before
screen shot 2016-07-12 at 11 49 37

after
screen shot 2016-07-12 at 11 48 52

also fixes table rows in the sortable state + adds integration tests for them

### CODE REVIEW

File: djangocms_admin_style/sass/components/_changelist.scss [hunk 1]
```diff
@@ -411,7 +411,7 @@
                 padding: 0 15px !important;
             }
             .text {
-                float: left;
+                float: none;
```
Comment (dhorelik) [inline djangocms_admin_style/sass/components/_changelist.scss]: can we just remove this or does it override a different float?

**Comment by dhorelik** [comment]
overall lgtm

File: djangocms_admin_style/sass/components/_changelist.scss (hunk 1 above)
Comment (vxsx) [inline djangocms_admin_style/sass/components/_changelist.scss]: it's an override

**Comment by lory87** [comment]
lgtm

**Pull request merged**